shared/oae/api/oae.api.util.js

```diff
@@ -1655,9 +1655,29 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
             if (!input) {
                 return '';
             } else {
-
+                var nonWord = /\W/;
                 // Escape asterisks instead of treating them as italics markers
-                input = input.replace(/\*/g, "\\*").replace(/\\\\/g, "\\");
+                input = input.replace(/\*/g, function(match, index) {
```

Comment:
I think the rules we want to follow can be captured in a regex:

For every line:

- all `*`s at the beginning of the line or `*`s preceded by a space
- and followed by a character different than a space
- followed by a different `*` preceded by a character different than a space
- and followed by a space or the end of the line

For every such combination of `*`s, the content should be made italic. All other `*`s should be escaped.